desktop/apps/auction/components/filter_sort/index.js

```diff
@@ -42,6 +44,12 @@ function FilterSort(props) {
   )
 }
 
+FilterSort.propTypes = {
```

Comment:
What are the chances we could start using Typescript and stop using prop types? (not in this PR obviously)